doc/api/next_api_changes/removals/18069.DS.rst

```diff
@@ -0,0 +1,5 @@
+Removed support for single color strings in `~.colors.to_rgba_array`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Converting a string with single color characters (e.g. ``'rgb'``) in
```

Comment:
Maybe use another sequence that does not coincidentially match the function name part "rgb(a)".